src/app/event/components/location/location.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="location">
+    <div class="location__name">
+        Location
+    </div>
+    <div class="location__map">
+        <iframe [src]="this.sanitizer.bypassSecurityTrustResourceUrl((event | async)?.location.url)"  width="600" height="300" frameborder="0" style="border:0;" allowfullscreen=""></iframe>
```

Comment:
same here about Elvis Operator

Reply:
Also, do not call methods in the template. It's performance ineffective since the function is called several times